nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,235 @@
+# Git as a Data Source
+
+The Nautobot Git as a Data Source feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot.  While using the benefits that Github already provides with issue tracking, discussions, pipelines, and approvals.
+
+For more in depth details on how this feature [see](https://nautobot.readthedocs.io/en/latest/models/extras/gitrepository/).
+
+
+## Supported Providers
+The feature uses the concept of a `provides` field to map a repository to a use case. A list of the supported options is provided below.
+
+|Name|Summary|Related Plugin|
+|:--|:--|:--|
+|Export Templates|Nautobot allows users to define custom templates that can be used when exporting objects.|N/A|
+|Jobs|Jobs are a way for users to execute custom logic on demand from within the Nautobot UI. Jobs can interact directly with Nautobot data to accomplish various data creation, modification, and validation tasks.|N/A|
+|Config Contexts||[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
+|Backup Configs|Backup configuration data.|[Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config)|
```

Comment:
I think you can remove these, imho, docs should only relate to core, but will defer to core committers.

Reply:
I'm personally not opposed to listing the plugins, but I want to make sure it's done in a way that doesn't make it seem like our plugins are the only way to do it.

Reply:
Added `core functionality` to be more explicitly on the data sources that come natively.  If different terminology is wanted let me know.

Reply:
I've changed the table into two tables.